frontend/src/components/SocialMedia.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div id="media">
+        <h2>MEDIA</h2>
+        <ul>
+            <li v-for="item in media" :key="item.link">
+                <a :href="item.link" target="_blank" class="circle">
+                    <v-icon> {{item.icon}} </v-icon>
+                </a>
+            </li>
+        </ul>
+    </div>
+</template>
+
+<script>
+export default {
+    data () {
+        return {
+            media : [
+                {
+                    icon: 'fab fa-github',
+                    link: 'https://github.com/CodeForPoznan',
+                },
+                {
+                    icon: 'fab fa-linkedin-in',
+                    link: 'https://www.linkedin.com/company/codeforpoznan/',
+                },
+                {
+                    icon: 'fab fa-facebook-f',
+                    link: 'https://www.facebook.com/CodeForPL/',
+                },
```

Comment:
Previously there was also gitter icon here and as we discussed last week since we don't use we shall skip it then. But we also found out that there is an easy way to open the Slack workspace so you might add the icon for Slack and referenced it with this link: https://join.slack.com/t/codeforpoznan/shared_invite/enQtNjQ5MTU1MDI0NDA0LWJmMmM2NjI0MGNiZjJjMGM0NzMzOTEzOGY5YWZkZWNiMzZlZTM2ZGJlYzRkYTlmZTFlMTMzZjJkZTc5YjhhYjg (or some shortened version of it)

Reply:
Okay, I'll do it :)